processor/src/test/java/org/mapstruct/ap/test/unmappedtarget/beanmapping/BeanMappingIgnoreTargetMapper.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.unmappedtarget.beanmapping;
+
+import org.mapstruct.BeanMapping;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public interface BeanMappingIgnoreTargetMapper {
+
+    BeanMappingIgnoreTargetMapper INSTANCE =
+            Mappers.getMapper( BeanMappingIgnoreTargetMapper.class );
+
+    @BeanMapping( ignoreTargets = {"nested.flag", "address"} )
```

Comment:
Shouldn't this also lead to: Target property "address" must not be mapped more than once.
```java
    @BeanMapping( ignoreTargets = {"nested.flag", "address", "nested.flag"} )
``` 

```java
    @Mapping( target = "address", ignore = true)
    @Mapping( target = "address", ignore = true)
``` 
does